fabric-data-attachment-api-v1/src/testmod/java/net/fabricmc/fabric/test/attachment/PersistentAttachmentTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.attachment;
+
+import com.mojang.serialization.Codec;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.util.Identifier;
+import net.minecraft.world.chunk.WorldChunk;
+
+import net.fabricmc.api.ModInitializer;
+import net.fabricmc.fabric.api.attachment.v1.AttachmentRegistry;
+import net.fabricmc.fabric.api.attachment.v1.AttachmentType;
+import net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents;
+
+public class PersistentAttachmentTest implements ModInitializer {
+	public static final String MOD_ID = "fabric-data-attachment-api-v1-testmod";
+	public static final Logger LOGGER = LoggerFactory.getLogger(MOD_ID);
+
+	private boolean firstLaunch = true;
+
+	@Override
+	public void onInitialize() {
+		AttachmentType<String> dummy = AttachmentRegistry.createPersistent(
```

Comment:
Nit: the test mods will likely be used as an example, I would make the attachment a public static final field to better match how its likely going to be used.